extension/_locales/en_US/messages.json

```diff
@@ -1,14 +1,14 @@
 {
     "extensionName": {
       "message": "Firefox Translations",
-      "description": "Name of the extension"
+      "description": "Name of the extension. Do NOT translate, keep the name in English."
```

Comment:
@flodolo @abhi-agg I originally thought the name of the extension should also be translated to the user's language.

Reply:
For reference, all the other add-ons we have remain in English (e.g. "Firefox Multi-account Containers"). 

But, in the end, it's a product call.

Reply:
We don't have product support at the moment, so let's follow the other add-ons policy then.

Reply:
Resolving this one.